### PR TITLE
Update changelog for latest release

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
cc/ @desktop/core

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+      "[Improved] Tidy up commit summary and description layout in commit list - #3922. Thanks @willnode!",
+      "[Improved] Use smaller default size when rendering Gravatar avatars - #3911",
+      "[Improved] Add a small amount of space between commit summary and description - #3922. Thanks @willnode!",
+      "[Improved] Show fetch progress when initializing remote for fork - #3953",
+      "[Improved] Remove references to Hubot from the user setup page - #4015. Thanks @j-f1!",
+      "[Improved] Error handling around ENOENT - #3954",
+      "[Improved] Clear repository filter text when switching tabs - #3787. Thanks @reyronald!",
+      "[Improved] Disable drag-and-drop interaction when a popup is in the foreground - #3996",
+      "[Improved] Allow window to accept single click on focus - #3843",
+      "[Added] Add RubyMine support for macOS - #3883. Thanks @gssbzn!",
```
[the remark this answers is not in the document]
Comment (iAmWillShepherd) [reply changelog.json]: Will do, but I ordered according to our [doc](https://github.com/desktop/desktop/blob/master/docs/process/releasing-updates.md).

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+      "[Improved] Update branch button text from \"New\" to \"New Branch\" - #4032",
+      "[Improved] Add fuzzy search in the repository, branch, PR, and clone FilterLists - #911. Thanks @j-f1!",
+      "[Improved] Tidy up commit summary and description layout in commit list - #3922. Thanks @willnode!",
+      "[Improved] Use smaller default size when rendering Gravatar avatars - #3911",
+      "[Improved] Add a small amount of space between commit summary and description - #3922. Thanks @willnode!",
+      "[Improved] Show fetch progress when initializing remote for fork - #3953",
+      "[Improved] Remove references to Hubot from the user setup page - #4015. Thanks @j-f1!",
+      "[Improved] Error handling around ENOENT - #3954",
+      "[Improved] Clear repository filter text when switching tabs - #3787. Thanks @reyronald!",
+      "[Improved] Disable drag-and-drop interaction when a popup is in the foreground - #3996",
```
Comment (shiftkey) [inline changelog.json]: Actually this is a dupe of line 15

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+    "1.1": [
+      "[New] Check out pull requests from collaborators or forks from within Desktop",
+      "[New] View the commit status of the branch when it has an open pull request",
+      "[Added] Add RubyMine support for macOS - #3883. Thanks @gssbzn!",
+      "[Added] Add TextMate support for macOS - #3910. Thanks @caiofbpa!",
+      "[Added] Syntax highlighting for Elixir files - #3774. Thanks @joaovitoras!",
+      "[Fixed] Layout of branch blankslate image - #4011",
+      "[Fixed] Expanded avatar stack in commit summary gets cut off - #3884",
+      "[Fixed] Autocomplete list for co-authors shows items that have already been matched - #3887",
+      "[Fixed] Branch blank slate image out of position - #4011",
```
Comment (shiftkey) [inline changelog.json]: Duplicate of line 9 - but I like this one better :fire:

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+      "[Fixed] Clear repository filter when switching tabs - #3787. Thanks @reyronald!",
```
Comment (shiftkey) [inline changelog.json]: Duplicate of line 31 - and I like the other one better

**Pull request merged**